hugegraph-core/src/main/java/com/baidu/hugegraph/task/HugeTaskScheduler.java

```diff
@@ -240,6 +268,23 @@ public boolean close() {
                   "Task '%s' was not completed in %s seconds", id, seconds));
     }
 
+    private <V> boolean completed(HugeTask<V> task) {
+        if (!task.completed()) {
+            return false;
+        }
+        // task itself is completed
+        if (task.children() == null || task.children().isEmpty()) {
+            return true;
+        }
+        for (Id id : task.children()) {
```

Comment:
prefer to update status of patent task when a task is completed, and just return status status of task itself when query task status